sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/implementation/models/ArtifactManifestProperties.java

```diff
@@ -104,6 +104,18 @@
     @JsonProperty(value = "manifest.changeableAttributes.readEnabled")
     private Boolean readEnabled;
 
+    /*
+     * Quarantine state
+     */
+    @JsonProperty(value = "manifest.changeableAttributes.quarantineState")
```

Comment:
None of these `.` including property names are flattened, right?

Reply:
Not sure I understand this question.